### ISSUE TITLE
Logging overrides global logging behaviour

### ISSUE THREAD

**Issue by fbordignon** [PMEAL/OpenPNM#2530]
https://github.com/PMEAL/OpenPNM/blob/8e2f1a400e5563bd55ff6823c8e80c32fc5e6b80/openpnm/utils/_workspace.py#L46
Isn't this line overriding the global root logger level?

This line should be enough to get the logger for the module:
https://github.com/PMEAL/OpenPNM/blob/8e2f1a400e5563bd55ff6823c8e80c32fc5e6b80/openpnm/utils/_workspace.py#L9

**Comment by fbordignon** [comment]
```python
>>> import logging
>>> logging.getLogger().level
30
>>> logging.getLogger().setLevel(10)
>>> logging.getLogger().level
10
>>> import openpnm
>>> logging.getLogger().level
30
```

**Comment by jgostick** [comment]
closed by PR #2531
